lib/membrane/video_compositor/implementation/opengl/native/rust.ex

```diff
@@ -63,7 +63,8 @@ end
 
 defmodule Membrane.VideoCompositor.Implementations.OpenGL.Native.Rust.Position do
   @moduledoc """
-  A Position struct describing the video position for use with the rust-based compositor implementation
+  A Position struct describing the video position for use with the rust-based compositor implementation.
+  Position relative to the top right corner of the background video, in the pixels.
```

Comment:
Maybe this can be rephrased as "Position relative to the top right corner of the viewport, in pixels" or "Position relative to the top right corner of the output, in pixels"